packages/kaizen-tg/_templates/drupal-9-theme/new/packages/components/_typography.css

```diff
@@ -1,6 +1,19 @@
 ---
```

Comment:
I can't add comment for empty files, so writing it here - 
ExampleFont.woff and ExampleFont.woff2 looks useless. `Fonts` directory means already you have to store fonts inside.
Also you can skip using `woff` already and use only `woff2`.